internal/host/validator.go

```diff
@@ -1032,3 +1032,40 @@ func printIsDomainNameResolvedCorrectly(c *validationContext, status ValidationS
 		return "Unexpected status"
 	}
 }
+
+func (v *validator) isDNSWildcardNotConfigured(c *validationContext) ValidationStatus {
+	if !isNonePlatformMultiNodeInstallation(c) {
```

Comment:
@eranco74 do we want [this](https://issues.redhat.com/browse/MGMT-6911) only on none-platform, or all the time?

Reply:
@eranco74 also please take a look at this PR in general, it's your ticket so you know best

Reply:
all the time, it effects all the platforms

Reply:
Great. So @bkhizgiy please remove this `if` statement and also don't forget to correct the commit message

Reply:
Ack. removed the check from the validation.

Reply:
Commit message + PR description still mention "user managed networking"